tests/modules/annotations/resources/test_matching_set.py

```diff
@@ -324,3 +325,109 @@ def test_annotation_matching_set_macros(
     assert len(replaced['filter']['C']['match']['should']) == len(vps)
     for vp in vps:
         assert {'term': {'viewpoint': vp}} in replaced['filter']['C']['match']['should']
+
+
+@pytest.mark.skipif(
+    module_unavailable('asset_groups'),
+    reason='AssetGroups module disabled',
+)
+@pytest.mark.skipif(
+    extension_unavailable('elasticsearch'),
+    reason='Elasticsearch extension or module disabled',
+)
+def test_search_with_elasticsearch(
+    flask_app_client,
+    researcher_1,
+    admin_user,
+    request,
+    test_root,
+):
+    # pylint: disable=invalid-name
+    from app.extensions import elasticsearch as es
+    from app.modules.annotations.models import Annotation
+
+    uuids = enc_utils.create_encounter(flask_app_client, researcher_1, request, test_root)
+    enc_guid = uuids['encounters'][0]
+    asset_guid = uuids['assets'][0]
+
+    tx = setting_utils.get_some_taxonomy_dict(flask_app_client, admin_user)
+    assert tx
+    assert 'id' in tx
+    taxonomy_guid = tx['id']
+    locationId = 'erehwon'
+    patch_data = [
+        utils.patch_replace_op('taxonomy', taxonomy_guid),
+        utils.patch_replace_op('locationId', locationId),
+    ]
+    enc_utils.patch_encounter(
+        flask_app_client,
+        enc_guid,
+        researcher_1,
+        patch_data,
+    )
+
+    viewpoint = 'upfront'
+    response = annot_utils.create_annotation(
+        flask_app_client,
+        researcher_1,
+        asset_guid,
+        enc_guid,
+        viewpoint=viewpoint,
+    )
+
+    annotation_guid = response.json['guid']
+    annotation = Annotation.query.get(annotation_guid)
+    request.addfinalizer(annotation.delete)
+    annotation.content_guid = uuid.uuid4()
+
+    # Index all users
```

Comment:
This should be `# Index all annotations`. The same line is copied in the other two elasticsearch test files too.

Reply:
Fixed